cmd/sponged/handlers/data.go

```diff
@@ -0,0 +1,55 @@
+// Package handlers contains the handler logic for processing requests.
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/web/app"
+	"github.com/coralproject/shelf/internal/sponge/item"
+)
+
+// defaultVersion is set to item.Version when no version is provided
+const defaultVersion = 1
+
+// dataHandle maintains the set of handlers for the data api, which is responsible
+// for all requests sending/requesting unstructured data not yet in item form.
```

Comment:
Sentence must start with the name of the type. Does not matter this is unexported.